app/js/models/addon_generator.js

```diff
@@ -126,25 +126,51 @@ el.parentNode.removeChild(el);
     );
   }
 
-  opSetAttribute(name, value) {
+  opCreateAttribute(expression, name) {
+    var assignment = (expression[0] === '[') ? `el${expression}` : `el.${expression}`;
+
+    this.operations.push(
+`/*=AddonGenerator::createAttribute*/
+${assignment}.setNamedItem(document.createAttribute(${JSON.stringify(name)}));
+/*==*/`
+    );
+  }
+
+  opCreateAttributes(attributes) {
+    for (var expression in attributes) {
+      this.opCreateAttribute(expression, attributes[expression]);
+    }
+  }
+
+  opRemoveAttribute(expression, name) {
```

Comment:
This isn't a real operation, it's just a helper function. It would be best to name it that way.